### ISSUE TITLE
My Response to Israeli Side

### ISSUE THREAD

**Issue by Zain-ul-din** [alisa-yar/The-Israeli-Side#1]
# Check out Palestine Side

- ## [Github Project link](https://github.com/Zain-ul-din/The-Palestinian-Side)
- ## [The Palestine Side](https://www.palestineside.site/)

**Comment by alisa-yar** [comment]
Thank you! And to be honest, eventually I came to the conclusion that both sides are bad and that we need more compassion and peace in our world.

**Comment by Zain-ul-din** [comment]
@alisa-yar I agree to some extent both nations have right to live. But if israeli army will kill Gaza kids there will certainly be a retaliatory response from the Palestine side and more people will join Hamas. 
Thank you! for your positive response.
